### PR TITLE
Add content preview in notifications

### PR DESCRIPTION
**Pull request by erditkurteshiSQA**
Fixes https://github.com/near/near-discovery/issues/583.

Based on design specifications:
https://www.figma.com/file/Cm6qATC1CpK9nUrKgx5K3U/Notifications?type=design&node-id=1-15248&mode=design&t=MdgBWoskxRaQuyj1-4

- Added styles to banner buttons.
- Added styles to the `See all` button in the notification tab.
- Added notification content preview.
- Replaced the `Turn on/off` button with `switcher` on the notification settings page.

### CODE REVIEW

File: src/NearOrg/Notifications/NotificationButton.jsx
Comment (shelegdmitriy) [inline src/NearOrg/Notifications/NotificationButton.jsx:166]:
```suggestion
                  variant: "primary",
                  fill: "outline",
```

File: src/NearOrg/Notifications/NotificationButton.jsx
Comment (shelegdmitriy) [inline src/NearOrg/Notifications/NotificationButton.jsx:174]:
```suggestion
                  style: {
                    width: "100%",
                  },
```

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]

File: src/NearOrg/Notifications/Banner.jsx
Comment (erditkurteshiSQA) [reply src/NearOrg/Notifications/Banner.jsx]: This has been addressed in the latest commit. Thank you for the review.

File: src/NearOrg/Notifications/Banner.jsx
Comment (erditkurteshiSQA) [reply src/NearOrg/Notifications/Banner.jsx]: This has been addressed in the latest commits.

File: src/NearOrg/Notifications/NotificationButton.jsx
Comment (erditkurteshiSQA) [reply src/NearOrg/Notifications/NotificationButton.jsx]: This has been addressed in the latest commits.

File: src/NearOrg/Notifications/NotificationButton.jsx
Comment (erditkurteshiSQA) [reply src/NearOrg/Notifications/NotificationButton.jsx]: This has been addressed in the latest commits.

File: src/NearOrg/Notifications/Notification.jsx
Comment (shelegdmitriy) [reply src/NearOrg/Notifications/Notification.jsx:233]: @erditkurteshiSQA have you checked this issue?

File: src/NearOrg/Notifications/Notification.jsx
Comment (shelegdmitriy) [inline src/NearOrg/Notifications/Notification.jsx:354]: This looks much cleaner:
```suggestion
        {actionable && contentDescription?.text && (
```

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]

File: src/NearOrg/Notifications/NotificationButton.jsx
Comment (erditkurteshiSQA) [reply src/NearOrg/Notifications/NotificationButton.jsx]: I realized that the button has a wrapper `div` element.
It should be good now with the latest change.
Thank you @shelegdmitriy

File: src/NearOrg/Notifications/SettingsListItem.jsx
Comment (shelegdmitriy) [inline src/NearOrg/Notifications/SettingsListItem.jsx:110]: I think it's better to use `checked` prop ([doc](https://www.radix-ui.com/primitives/docs/components/switch#root)) here instead of this conditions. And no need to use `data-state` for `Switch.Thumb` component

**Review by shelegdmitriy — APPROVED** [review]
LGTM! I left the last comment which doesn't block the preview of notifications so feel free to address it in separate PR

**Comment by charleslavon** [comment]
> LGTM! I left the last comment which doesn't block the preview of notifications so feel free to address it in separate PR

hey @shelegdmitriy Just a heads up, our new branch protection rules require that all conversations be resolved before. PR can be merged, so if you don't want that comment to block this PR please resolve it.

**Comment by shelegdmitriy** [comment]
> > LGTM! I left the last comment which doesn't block the preview of notifications so feel free to address it in separate PR
> 
> 
> 
> hey @shelegdmitriy Just a heads up, our new branch protection rules require that all conversations be resolved before. PR can be merged, so if you don't want your open conversations to block this PR please resolve them.

Yeah, I know that. @erditkurteshiSQA feel free to resolve that comment by your own

File: src/NearOrg/Notifications/SettingsListItem.jsx
Comment (erditkurteshiSQA) [reply src/NearOrg/Notifications/SettingsListItem.jsx:107]: After I close this one, I will create a separate PR for this suggestion. Thank you.

**Pull request merged**
